code/hsec-tools/app/Main.hs

```diff
@@ -44,6 +47,16 @@ commandCheck =
           die $ "Filename does not match advisory ID: " <> path
       T.putStrLn "no error"
 
+commandOsv :: Parser (IO ())
+commandOsv =
+  withAdvisory go
+  <$> optional (argument str (metavar "FILE"))
+  <**> helper
+  where
+    go _ adv = do
+      L.putStr (OSV.encode (OSV.convert adv))
```

Comment:
@blackheaven this is where the re-export pays off.  I didn't have to explicitly import `Data.Aeson.encode` to use it here.

If you still prefer to avoid the re-export and instead import `Data.Aeson` directly, I'm OK with that.  I can make the change and update the PR tomorrow.